heka/plugins/s3splitfile/s3splitfile_input.go

```diff
@@ -204,6 +212,16 @@ func (input *S3SplitFileInput) fetcher(runner pipeline.InputRunner, wg *sync.Wai
 	wg.Done()
 }
 
+func (input *S3SplitFileInput) ReportMsg(msg *message.Message) error {
+	message.NewInt64Field(msg, "ProcessFileCount", atomic.LoadInt64(&input.processFileCount), "count")
+	message.NewInt64Field(msg, "ProcessFileFailures", atomic.LoadInt64(&input.processFileFailures), "count")
+	message.NewInt64Field(msg, "ProcessMessageCount", atomic.LoadInt64(&input.processMessageCount), "count")
+	message.NewInt64Field(msg, "ProcessMessageFailures", atomic.LoadInt64(&input.processMessageFailures), "count")
+	message.NewInt64Field(msg, "ProcessMessageSumSize", atomic.LoadInt64(&input.processMessageSumSize), "B")
```

Comment:
The sandbox input uses ProcessMessageBytes.  We should select a standardized name.